
help: Configuration file, two robots, two targets, and one type II zone
testID: 103
exp: "simulation"  # 0: "simulation", 1: "ros simulation", 2: "ros real"
steps: 300

solver: "All" # "RiskAware" or "RiskNeutral" or "Dist" or "All"
frame_id: "world"

Problem:
    robotID: [0, 1]
    robotID_drone: [3, 4]
    targetID: [0, 1]
    targetID_car: [3, 4]
    targetStartPos: [[-1, 0.4], [-1, -0.4]] #[[-2, -0.3], [-2, 0.3]] #, [1.5, 0.25]]  # [x, y], or [x, y, z]
    targetStartVel: [[0.1, 0], [0.1, 0]] #, [-0.1, 0]]
    robotStartPos: [[-2.0, 0.6], [-2.0, -0.6]] #[[-4, -0.9], [-4, 0.9]] #, [2.0, 0.2]]
    robotStartVel: [[0.0, 0], [0, 0]] #, [0, 0]]
    target_dyn_type: "second_order_dynamics"
    robot_dyn_type: "first_order_dynamics" # "second_order_dynamics"
    task_assignment_matrix: [[1, 0], [0, 1]] # [0, 0, 1]]  # robot i is assigned to target j
    target_movement: "linear"  # "random_walk" or "linear"
    # row is robot, column is target

    N : 2
    dt: 0.1
    dim: 2
    u_bounds: [-0.5, 0.5]  # control effort bounds
    x_bounds: [[-2.2, 2.2], [-2.2, 2.2]] #[[-6, 6], [-6, 6]]  # x bound, and y bound
    weights : [0.05, 10.0, 100.0, 100.0] # control input, trace, slack 1, slack 2  # it's variables that you can change 
    range_sensor: [15.0, 0.2]   #[range_peak, range_shape]
    bearing_sensor: [15.0, 0.2]   #[bearing_peak, bearing_shape]
    min_dist: 0.8
    max_dist: 6000  # can also affect the quality of the solution, can set inf
    solver_name: "resilient"
    use_cent_solver: True # if True, the central solver will be used

    ## archived
    resources: 500 # for each robot

Zones:
    nTypeI: 0
    nTypeII: 1
    typeI_mu: [[2.5, 0.5]] # [x, y], or [x, y, z]     [[x1, y1], [x2, y2]]
    typeI_cov: [[0.3, 0.3]]
    typeII_mu: [[0.1, 0]] # [[-0.2, 0.0]]
    typeII_cov: [[0.3, 0.3]] #[[0.6, 0.6]]
    typeI_delta: []  # this is for the type I zones
    typeII_delta: [0.1, 0.1]  # this is for the type II zones
    eps1: 0.05
    eps2: 0.9
    eps2_single: 0.05

    #### attack parameters
    attack_recover_eps: 0.2
    attack_dt: 1 # every 1.0 second the attack will happen
    attack_mcnt: 1000 # the maximum number of attacks
    attack_seed: 0
